### ISSUE TITLE
DRBD Tuning

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#113]
Originally reported of Google Code with ID 57.

```
Hello!

Can anyone tell me if it is possible to do some tuning on drbd when using
ganeti? I can do this using the drbd-cli, but changes are overwriten by
ganeti if I understand correctly?
```
Originally added on 2009-04-20 10:00:51 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Unfortunately yes.

We're planning to add some more customization to DRBD, but we don't have
a roadmap for this yet.

regards,
iustin

```
Originally added on 2009-04-21 08:48:52 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2010-03-12 19:39:33 +0000 UTC.

Added to Milestone: Unplanned

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2011-08-05 14:45:42 +0000 UTC.

Removed from Milestone: Unplanned

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been implemented in the 2.6 release (which will come out later this month), via the new disk parameters functionality.
```
Originally added on 2012-07-19 12:51:44 +0000 UTC.

Added Labels: Type-Enhancement
Changed State: Fixed
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
